_posts/2017-12-11-away-week-prague/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "The importance of face-to-face in a remote world"
+title-short: Face-to-Face
+tags: [company culture, management]
+author: Anthony
+category: Inside the Plow
+permalink: /blog/2017/12/11/importance-of-face-to-face/
+description: "The role of team retreats for distributed companies"
+---
+
+![prague-scene][prague]
+
```

Comment:
This image isn't working on next.snowplowanalytics.com - is it because the image hasn't been uploaded?